docs/source/index.rst

```diff
@@ -1,72 +1,131 @@
-
-   - Cell toolbar selector moved to View menu
-   - Restart & Run All Cells added to Kernel menu
-   - Multiple-cell selection and actions including cut, copy, paste and execute
-   - Command palette added for executing Jupyter actions
-   - Find and replace added to Edit menu
-
-   To upgrade to the release:
-   ``pip install notebook --upgrade``
-   or
-   ``conda upgrade notebook``
-
-Using the Jupyter Notebook, you can author engaging
-documents that combine live-code with narrative text, equations, images, video,
-and visualizations. By encoding a complete and reproducible record of a
-computation, the documents can be shared with others on GitHub, Dropbox, and
-the `Jupyter Notebook Viewer <https://nbviewer.jupyter.org/>`__.
-
-The main documentation for Jupyter users is organized into several sections:
-
-    - :ref:`User Documentation<user-docs>`
-    - :ref:`Jupyter Subprojects<jupyter-subprojects>`
-    - :ref:`About Jupyter<about-docs>`
-
-:ref:`Contributor Documentation<dev-docs>` covers how to help with the
-development of Jupyter as well as technical details helpful for development. It
-also covers how to contribute to our documentation, blog, website, and
-newsletter.
-
+===========================
+Project Jupyter and IPython
+===========================
+
+The "official" documentation for **Project Jupyter and IPython.**
+
+**Parts of the documentation:**
+
+.. raw:: html
+
+  <table align="center"><tr>
+    <td width="50%">
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#user-docs">Jupyter Notebook Quickstart</a><br/>
+        <span class="linkdescr">Start here</span></p>
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#about-docs">User stories</a><br/>
+        <span class="linkdescr">Common use cases in industries</span></p>
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#jupyter-subprojects">Projects</a><br/>
+         <span class="linkdescr">High level descriptions</span></p>
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#project-links">Documentation by project</a><br/>
+         <span class="linkdescr">Usage, configuration, and development information</span></p>
+      </td><td width="50%">
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#tech-overview">Technical overview</a><br/>
+         <span class="linkdescr">Visual overview of how projects are used together</span></p>
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/#dev-docs">Contributor Guide</a><br/>
+         <span class="linkdescr">The general process of contributing code, documentation, community resources, and more</span></p>
+      <p><a href="https://test-jupyter.readthedocs.org/en/landing-twocol/development_guide/index.html">IPython Development guide</a><br/>
+         <span class="linkdescr">Legacy IPython wiki reference documentation</span></p>
+    </td></tr>
+  </table>
```

Comment:
I'm not sure we want to go this route of having raw Html in our docs, as it' a rabbit hole, and it will start not working great on small screen. I'm fine if there is consensus to do it, but I'm not for it.